src/main/java/slimeknights/tconstruct/tools/modifiers/upgrades/NetheriteModifier.java

```diff
@@ -21,9 +21,9 @@ public void addVolatileData(ToolDefinition toolDefinition, StatsNBT baseStats, I
 
   @Override
   public void addToolStats(ToolDefinition toolDefinition, StatsNBT baseStats, IModDataReadOnly persistentData, IModDataReadOnly volatileData, int level, ModifierStatsBuilder builder) {
-    builder.multiplyDurability(1.25f);
-    builder.multiplyAttackDamage(1.25f);
-    builder.multiplyMiningSpeed(1.25f);
+    builder.multiplyDurability(1.20f);
+    builder.multiplyAttackDamage(1.10f);
```

Comment:
10% might be a little low. Vanilla axes are 11%, and vanilla swords are 14%. Not sure if that is significant, so worth discussing on discord.